### PR TITLE
Fix issue #614: restart queue/evaluator on sufficient disk space avai…

### PR DESCRIPTION
**Pull request by kquick**
…lable.

### CODE REVIEW

File: hydra-module.nix [hunk 1]
```diff
@@ -396,17 +396,30 @@ in
...
-                echo "stopping Hydra queue runner due to lack of free space..."
-                systemctl stop hydra-queue-runner
-            fi
-            if [ $(($(stat -f -c '%a' /nix/store) * $(stat -f -c '%S' /nix/store))) -lt $((${toString cfg.minimumDiskFreeEvaluator} * 1024**3)) ]; then
-                echo "stopping Hydra evaluator due to lack of free space..."
-                systemctl stop hydra-evaluator
-            fi
+            spaceleft=$(($(stat -f -c '%a' /nix/store) * $(stat -f -c '%S' /nix/store)))
+            spacestopstart() {  # $1 = job, $2 = min GB free
+              if [ $spaceleft -lt $(($2 * 1024**3)) ]; then
```
Comment (grahamc) [inline hydra-module.nix:407]: Can you give useful names to `$1` and `$2` in the function? Also, can you print a message in the "starting" block?

Comment (kquick) [reply hydra-module.nix]: Done, @grahamc .

**Comment by grahamc** [comment]
Thank you!

**Comment by kquick** [comment]
Thanks for the merge, @grahamc.  You might want to close issue 614 now.

**Pull request merged**